src/v3/src/components/AuthCoin/AuthCoin.test.tsx

```diff
@@ -66,19 +70,21 @@ describe('AuthCoin tests', () => {
       url: '/img/socialButtonIcons/okta.svg',
     };
 
-    const { findByText } = render(<AuthCoin {...props} />);
+    const { container, findByRole } = render(<AuthCoin {...props} />);
 
-    expect(await findByText('', { selector: 'mocksvgicon' })).toBeDefined();
+    expect(await findByRole('img')).toBeDefined();
+    expect(container).toMatchSnapshot();
   });
 
   it('should not render the component when invalid authenticator key is provided', async () => {
```

Comment:
if the authenticator key doesn't match anything that the SIW knows about, it wont render anything. This scenario isnt really possible unless a new authenticator is introduced on the backend without SIW being notified (which I assume is highly unlikely).

Reply:
As OIE support grows this might happen but I believe for v2 parity we shouldn't yet be concerned about a very dynamic API. Something we _should_ have backstops for, though.